Source/Client/Windows/ServerBrowser.cs

```diff
@@ -344,10 +359,18 @@ private void DrawSaveList(List<SaveFile> saves, float width, ref float y)
                 else
                 {
                     GUI.color = saveFile.VersionColor;
-                    Widgets.Label(infoText.Down(16), (saveFile.rwVersion ?? "???").Truncate(110));
+                    Widgets.Label(infoText.Down(16), (saveFile.rwVersion ?? "???").Truncate(110));                        
                 }
 
-                if (saveFile.replay && saveFile.protocol != MpVersion.Protocol)
+                if (!saveFile.Valid)
```

Comment:
If save file is not valid, set a tooltip on the save file shown in the list of saves that mentions the save file is of unknown format. Looks good.